bom/pom.xml

```diff
@@ -61,6 +61,8 @@
     <truth.version>0.28</truth.version>
     <vertx.version>3.6.3</vertx.version>
     <wiremock.version>2.22.0</wiremock.version>
+    <docker.repository>docker.io</docker.repository>
+    <docker.image.additional.tag></docker.image.additional.tag>
```

Comment:
I guess these two can be removed, or what are they needed for?

Reply:
Correct